user-registration-app/src/app/services/resolvers/RegistrationResolver/registration.resolver.ts

```diff
@@ -37,7 +37,7 @@ export class RegistrationResolver implements Resolve<Registration | Registration
             this.router.navigate([AppConstants.PIN_ENDPOINT]);
             return null;
           }
-          return registration;
+          return registration.parseFromApiResponse(registration);
```

Comment:
return Registration.parseFromApiResponse(registration) after the method is made static

Reply:
done